apps/financial-aid/api/src/app/modules/amount/amount.resolver.ts

```diff
@@ -0,0 +1,30 @@
+import { Resolver, Context, Mutation, Args } from '@nestjs/graphql'
```

Comment:
I am going back and forth if we want to create an amount resolver. Wondering if it is an overkill instead of having it in applications like we did for aid, any thoughts?